src/neptune_detectron2/impl/__init__.py

```diff
@@ -142,13 +148,13 @@ def _log_checkpoint(self, final: bool = False) -> None:
         checkpoint_path = self.trainer.checkpointer.get_checkpoint_file()
 
         with open(checkpoint_path, "rb") as fp:
-            self._run[neptune_model_path] = File.from_stream(fp)
+            self._root_object[neptune_model_path] = File.from_stream(fp)
```

Comment:
I think this should stay `self._run` and preserve the base namespace.

Reply:
Ok, makes sense